src/Beta/Model/AccessLevel.php

```diff
@@ -12,7 +12,7 @@
 * @version   GIT: 1.4.0
 * @link      https://graph.microsoft.io/
 */
-namespace Microsoft\Graph\Model;
```

Comment:
@MIchaelMainer Since this is a beta library, the namespace change isn't considered a breaking change, right?

Reply:
I'm not sure what happened there. It should have been beta all along. I've got a lot of merge conflicts to go through on this one. This is on hold for now.